### DISCUSSION TITLE
Will the upload speed of files in MinIO be limited by the read/write speed of the /tmp directory?

### DISCUSSION THREAD

**Discussion by FourSpaces** [minio/minio#18966] [Q&A] [answered]
Will the upload speed of files in MinIO be limited by the read/write speed of the /tmp directory? For example, if my /tmp directory is an HDD disk with a read/write speed of only 150MB/s, will the file upload speed in MinIO exceed 150MB/s?

**Comment by klauspost** [comment, marked as answer]
No. MinIO only uses the provided storage for uploads. It will not use `/tmp`.
